pkg/provider/handlers/deploy.go

```diff
@@ -247,7 +247,11 @@ func prepareEnv(envProcess string, reqEnvVars map[string]string) []string {
 }
 
 func getMounts() []specs.Mount {
-	wd, _ := os.Getwd()
+	wd, set := os.LookupEnv("HOSTS_DIR")
+	if !set {
+		wd, _ = os.Getwd()
+		wd = path.Join(wd, "..", "faasd")
```

Comment:
How would you consume `HOSTS_DIR`? Perhaps this should be set explicitly in the systemd unit file to `/var/lib/faasd-provider`

Reply:
I would look for an entry in the service such as `faasd_hosts_file`

https://github.com/openfaas/faasd/blob/master/hack/faasd-provider.service